src/pages/EmptyPage/EmptyPage.module.css

```diff
@@ -0,0 +1,34 @@
+.empty_collection_container {
+  margin: 1rem;
+}
+
+.empty_collection_contents {
+  gap: 1rem;
```

Comment:
Just a suggestion gap is not supported in some browsers try achieving this with flex only,

Reply:
Yes I have used flex gap property itself . I used utility classes of flex itself for that div